index.bs

```diff
@@ -10125,6 +10132,86 @@ The [=remote end steps=] given |session|, and |command parameters| are:
 
 </div>
 
+#### The input.setFiles Command ####  {#command-input-setFiles}
+
+The <dfn export for=commands>input.setFiles</dfn> command sets the <code>files</code> property of a given <code>input<code> element with type <code>file</code>
+to a set of file paths.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      input.SetFiles = (
+        method: "input.setFiles",
+        params: input.SetFilesParameters
+      )
+
+      input.SetFilesParameters = {
+        context: browsingContext.BrowsingContext,
+        element: script.SharedReference,
+        files: [*text]
+      }
+    </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl">
+     EmptyResult
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for input.setFiles">
+
+The [=remote end steps=] given |session| and |command parameters| are:
+
+1. Let |context id| be the value of the |command
+   parameters|["<code>context</code>"] field.
+
+1. Let |context| be the result of [=trying=] to [=get a browsing context=] with
+   |context id|.
+
+1. Let |document| be |context|'s [=active document=].
+
+1. Let |environment settings| be the [=environment settings object=] whose
+   [=relevant global object=]'s <a>associated <code>Document</code></a> is
+   |document|.
+
+1. Let |realm| be |environment settings|'s [=realm execution context=]'s
+   Realm component.
+
+1. Let |element| be the result of [=trying=] to [=deserialize remote reference=]
+   with |command parameters|["<code>element</code>"], |realm|, and |session|.
+
+1. If |element| doesn't implement {{HTMLInputElement}}, return [=error=] with [=error code=]
```

Comment:
I actually wonder if we should only fail if the element doesn't implement `Element` here, and do the input check together with type `file` in the next step. I feel that this would make more sense given that we actually might have found an element but not with the right type.

Reply:
Makes sense. Done.